### PR TITLE
Update images

### PR DESCRIPTION
**Pull request by rnorth**
... So that workshop attendees can fetch most of the required images in one go

### CODE REVIEW

File: src/acceptanceTest/java/com/example/demo/SystemTest.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.utility.MountableFile;
+
+import java.io.File;
+import java.util.stream.Stream;
+
+/**
+ * TODO: Javadocs
+ */
+public class SystemTest {
```
Comment (bsideup) [inline src/acceptanceTest/java/com/example/demo/SystemTest.java]: pushed by accident?

Comment (rnorth) [reply src/acceptanceTest/java/com/example/demo/SystemTest.java]: Damnit, thanks. Fixed.

**Comment by bsideup** [comment]
Assigned @kiview as a reviewer since he added this code initially

File: step-9-selenium-tests.md [hunk 2]
```diff
@@ -61,7 +61,7 @@ public class SystemTest {
             .withNetworkAliases("kafka");
 
     @ClassRule
-    public static BrowserWebDriverContainer browser = (BrowserWebDriverContainer) new BrowserWebDriverContainer()
+    private static BrowserWebDriverContainer browser = new BrowserWebDriverContainer<>()
```
Comment (kiview) [inline step-9-selenium-tests.md]: Doesn't `@ClassRule` need the public modifier?

Comment (rnorth) [reply step-9-selenium-tests.md]: Crikey, you're right. #16 will replace, but I'll change anyway.

Comment (rnorth) [inline step-9-selenium-tests.md]:
```suggestion
    public static BrowserWebDriverContainer browser = new BrowserWebDriverContainer<>()
```

File: step-9-selenium-tests.md [hunk 3]
```diff
@@ -41,7 +41,7 @@ public class SystemTest {
             .withNetwork(net)
             .withNetworkAliases("redis");
 
-    private static GenericContainer sut = new GenericContainer("java:8-jre-alpine")
+    private static GenericContainer sut = new GenericContainer("openjdk:8-jre-alpine")
```
Comment (rnorth) [inline step-9-selenium-tests.md]:
```suggestion
    public static GenericContainer sut = new GenericContainer("openjdk:8-jre-alpine")
```

File: step-9-selenium-tests.md [hunk 4]
```diff
@@ -29,7 +29,7 @@ public class SystemTest {
 
     private static Network net = Network.newNetwork();
 
-    private static PostgreSQLContainer postgres = (PostgreSQLContainer) new PostgreSQLContainer()
+    private static PostgreSQLContainer postgres = new PostgreSQLContainer<>("postgres:10-alpine")
```
Comment (rnorth) [inline step-9-selenium-tests.md]:
```suggestion
    public static PostgreSQLContainer postgres = new PostgreSQLContainer<>("postgres:10-alpine")
```

**Pull request merged**